src/state/sagas.js

```diff
@@ -123,7 +130,22 @@ export async function fetchAnnotationResource(url) {
 }
 
 /** Saga for fetching external annotation resources */
-export function* fetchExternalAnnotationResources({ targetId, annotationId, annotationJson }) {
+export function fetchExternalAnnotationResources({ targetId, annotationId, annotationJson }) {
```

Comment:
Are you sure that this is working as expected? This should be a Generator that yields Redux-Saga effects and not a plain function.

Reply:
Thank you for pointing this out. I did thorougly test the `seeAlso` variant and only checked the "inline" one.